python/packages/streamsx/rest.py

```diff
@@ -48,23 +49,38 @@ def __init__(self, username=None, password=None, resource_url=None, config=None)
         if username and password and resource_url:
             self.rest_client = StreamsRestClient(username, password, resource_url)
             self.resource_url = resource_url
+            self._analytics_service = False
 
         # Connect to Bluemix service using VCAP
         elif config:
             vcap_services = VcapUtils.get_vcap_services(config)
             credentials = VcapUtils.get_credentials(config, vcap_services)
+            self._analytics_service = True
 
             # Obtain the streams SWS REST URL
             rest_api_url = VcapUtils.get_rest_api_url_from_creds(credentials)
 
             # Create rest connection to remote Bluemix SWS
             self.rest_client = StreamsRestClient(credentials['userid'], credentials['password'], rest_api_url)
             self.resource_url = rest_api_url
+
+        elif username and password and st._has_local_install:
+            self.resource_url = st.get_rest_api()
```

Comment:
Shouldn't we ensure that the `resource_url` provided in the call to `__init__` and the `resource_url` returned from `st.get_rest_api()` are the same? We don't want to end up cancelling jobs from Streams install A when we meant to cancel jobs from Streams install B